chore: Add section about error variants to contribution guide. (#5343)
dfinity/ic: rs/execution_environment/Contributing.md

4. [Motoko](https://github.com/dfinity/motoko) needs to be updated. Inform *@eng-motoko* of the work required.

> 5. Add any new types or update existing ones in `types/management_canister_types`. Where adequate, stick to existing naming conventions. E.g., error variants tend to have a domain specific prefix and a clear suffix describing the error. 

Reviewer: Could you please add an example (of an existing error variant) to illustrate your point?

> 5. Add any new types or update existing ones in `types/management_canister_types`. Where adequate, stick to existing naming conventions. E.g., error variants tend to have a domain specific prefix and a clear suffix describing the error. 

Reviewer: ```suggestion
5. Add any new types or update existing ones in `types/management_canister_types`. If possible, stick to existing naming conventions. E.g., error variants tend to have a domain specific prefix and a clear suffix describing the error. 
```
